crates/api/src/kitsune.rs

```diff
@@ -76,8 +79,11 @@ pub trait Kitsune: 'static + Send + Sync + std::fmt::Debug {
 
     /// Get an existing space with the provided [SpaceId] or create
     /// a new one.
-    fn space(&self, space_id: SpaceId)
-        -> BoxFut<'_, K2Result<space::DynSpace>>;
+    fn space(
```

Comment:
Missing documentation about the new parameter. Should probably match the docs above